langchain/src/vectorstores/tests/xata.int.test.ts

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable no-process-env */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { BaseClient } from "@xata.io/client";
+
+import { XataVectorSearch } from "../xata.js";
+import { OpenAIEmbeddings } from "../../embeddings/openai.js";
+import { Document } from "../../document.js";
+
+test("XataVectorSearch integration", async () => {
```

Comment:
Can you add `.skip` here? We don't typically run every integration test for our pre-release checks.